### DISCUSSION TITLE
Indexing Files Using viosay/conan-embedding-v1 Model

### DISCUSSION THREAD

**Discussion by Chinaduanyun** [logancyang/obsidian-copilot#1176] [Q&A]
Hello,

I hope this message finds you well. I am using the Copilot plugin in Obsidian for my knowledge management, and I have encountered an issue that I hope you can assist with.

I am utilizing the `viosay/conan-embedding-v1` model for indexing and retrieval. However, I've noticed that the "Total files indexed" count is displaying only 50, despite having 200 articles in my vault. This is in contrast to when I use the `mxbai-embed-large` model, which successfully indexes all 200 articles without issue.

I have ensured that the indexing process with the `viosay/conan-embedding` model has been completed, but the number of files being indexed is not reflecting the actual count. I am confident that there might be a specific reason or a setting that I might have overlooked.

Could you please provide some guidance on how to troubleshoot this issue or if there are any known limitations or issues with the `viosay/conan-embedding-v1` model that I should be aware of?

Thank you very much for your time and assistance. I look forward to your response.

Best regards,

**Comment by logancyang** [comment]
Do you have the same setting except the embedding model for both cases, such as inclusion/exclusion etc.

A good way to test is to reset settings to default, only change the embedding models and test again. I don't see a particular reason why embedding model can affect number of files.
